controllers/profile_controller.go

```diff
@@ -95,7 +95,13 @@ func (r *ProfileSubscriptionReconciler) Reconcile(ctx context.Context, req ctrl.
 		return ctrl.Result{}, err
 	}
 
-	pDef, err := git.GetProfileDefinition(pSub.Spec.ProfileURL, pSub.Spec.Branch, logger)
+	//TODO add validation for arguments in PCTL
```

Comment:
Shouldn't we add them here too? Do you expect PCTL to ever be the only consumer / user?

Reply:
I think we should do that here since what if someone just updates stuff by hand?

Reply:
but what will there be to update :smile: the profile subscription will be gone